documentation/carving/carving.md

```diff
@@ -128,48 +127,53 @@ These additional options described below can be displayed by scrolling down
 in the labeling applet on the left side.
 
 - **BG priority** The bias is a parameter the affects how much the background is
-  preferred in comparison to the other labels. A value smaller then 1.0 will
+  preferred in comparison to the other labels. A value smaller than 1.0 will
   lower the detected boundaries for the background seeds. Since the normal
   seeds still work on the original boundaries the background is preferred in
-  case of ambiguity. Usually a value of around 0.95 yields good results, but
-  sometimes playing with the parameter is a good way to improve segmentations
+  case of ambiguity. Usually, a value of around 0.95 yields good results, but
+  sometimes playing with the parameter is a good way to improve segmentation
   without additional seeds.
 - **No bias below** The threshold is a value that affects when the **BG priority**
   for the background will be applied. Normally the background seed is only
   preferred when the boundaries are sufficiently strong, i.e. > 64 (the
-  boundaries in the image have values between 0 and 255). Usually it is not
+  boundaries in the image have values between 0 and 255). Usually, it is not
   necessary to change this parameter.
   
-<a href="screenshots/carving-4.png" data-toggle="lightbox"><img src="screenshots/carving-4.png" class="img-responsive" /></a>
+<a href="screenshots/carving-04-labeling-options.png" data-toggle="lightbox"><img src="screenshots/carving-04-labeling-options.png" class="img-responsive" /></a>
 
 
 ## Saving and loading segmented objects
 
-Once the user has successfully segmented an object, the segmentation result
+Once the user has successfully segmented an object, the result
 can be stored by clicking on the **Save As** button. A dialog will pop up that
-asks for the objects name:
+asks for the object name, by default autogenerated name will be provided based on value of **Object name prefix** field:
```

Comment:
Maybe just mention that this autogeneration appends numbers to the object prefix

Reply:
reworded a bit